layouts/partials/header.html

```diff
@@ -12,7 +12,7 @@
         {{ with .Site.Menus.main}}
           {{ range sort . }}
             <li class="navigation-item">
-              <a class="navigation-link" href="{{ index (apply (slice .URL) (.Params.urlFunc | default "relLangURL") ".") 0 }}">{{ .Name }}</a>
+              <a class="navigation-link {{ .Params.class }}" href="{{ index (apply (slice .URL) (.Params.urlFunc | default "relLangURL") ".") 0 }}">{{ .Name }}</a>
```

Comment:
Let me just ask for one more thing. Could you add an entry in the `docs` folder about this new configuration parameter?

Reply:
@luizdepra, I've added a entry in the docs with commit 28d4538.
Is that good enough or you want me to extend that even further?

Reply:
LGTM. Ty. :)